latest/index.bs

```diff
@@ -241,9 +249,28 @@ keys as specified below for discovering certain types of data, especially images
 
 If part of [[#multiscale-md]], the length of "axes" MUST be equal to the number of dimensions of the arrays that contain the image data.
 
+"bioformats2raw.layout" (transitional) {#bf2raw}
+------------------------------------------------
+
+[=Transitional=] "bioformats2raw.layout" metadata identifies a group which implicitly describes a series of images.
+The need for the collection stems from the common "multi-image file" scenario in microscopy. Parsers like Bio-Formats
+define a strict, stable ordering of the images in a single container that can be used to refer to them by other tools.
+In order to capture that information within an OME-NGFF dataset, `bioformats2raw` internally introduced a wrapping
+layer. An upcoming NGFF specification will replace this layout with explicit metadata.
+
+Conforming groups:
+- MUST have the value "3" for the "bioformats2raw.layout" key at the top of the hierarchy.
```

Comment:
Does this mean that the `.zattrs` has `"bioformats2raw.layout": 3`?
An example layout would help I think.